src/schema/v2/artwork/index.ts

```diff
@@ -1872,10 +1880,67 @@ export const ArtworkConnectionInterface = new GraphQLInterfaceType({
   },
 })
 
+const PartnerOfferSorts = {
+  type: new GraphQLEnumType({
+    name: "PartnerOfferSorts",
+    values: {
+      CREATED_AT_ASC: {
+        value: "created_at",
+      },
+      CREATED_AT_DESC: {
+        value: "-created_at",
+      },
+      END_AT_ASC: {
+        value: "end_at",
+      },
+      END_AT_DESC: {
+        value: "-end_at",
+      },
+    },
+  }),
+}
+
 export const artworkConnection = connectionWithCursorInfo({
   nodeType: ArtworkType,
   connectionInterfaces: [ArtworkConnectionInterface],
   edgeInterfaces: [ArtworkEdgeInterface],
+  edgeFields: {
+    partnerOffers: {
+      type: connectionWithCursorInfo({
+        nodeType: PartnerOfferType,
+      }).connectionType,
+      args: pageable({
+        page: { type: GraphQLInt },
+        size: { type: GraphQLInt },
+        sort: PartnerOfferSorts,
+      }),
+      resolve: async (root, args: CursorPageable, { partnerOffersLoader }) => {
+        if (!partnerOffersLoader) return
```

Comment:
This works, sometimes you can do `throw new Error("You must be logged-in")` to be more informative to a client (they're missing an access token), vs. silently returning `null`

Reply:
- I moved the partnerOffersConnection to the Artwork type
- It now throws an error ("You need to be signed in to perform this action") instead of returning null